src/fabric/src/fabric2_db.erl

```diff
@@ -1289,6 +1380,25 @@ check_duplicate_attachments(#doc{atts = Atts}) ->
     end, ordsets:new(), Atts).
 
 
+get_since_seq(Seq) when Seq == <<>>; Seq == <<"0">>; Seq == 0->
+    fabric2_util:seq_zero_vs();
+
+get_since_seq(Seq) when Seq == now; Seq == <<"now">> ->
+    fabric2_util:seq_max_vs();
```

Comment:
I don't think this is correct. Seq "now" used to stand for db's max update seq, not ever possible max update seq. A changes feed with since `seq_max_vs` will ever receive updates.

Reply:
Aha. I do believe you are correct. Must've confused myself handling the endkey side of things.

Reply:
Fixed.